### PR TITLE
Make people go to Space

### PR DESCRIPTION
**Pull request by chr0nu5**

### CODE REVIEW

File: bot_modules/spaces.js [hunk 1]
```diff
@@ -1,13 +1,68 @@
...
+            phrase = randomPhrases.day[Math.floor(Math.random() * randomPhrases.day.length)];
+        } else if (when == "pela manha" || when == "pela manhã") {
+            phrase = randomPhrases.morning[Math.floor(Math.random() * randomPhrases.day.length)];
+        } else if (when == "a tarde" || when == "à tarde") {
+            phrase = randomPhrases.afternoon[Math.floor(Math.random() * randomPhrases.day.length)];
+        } else {
+            phrase = defaultPhrase;
+        }
+        if (phrase !== defaultPhrase) {
+            var _this = this;
```
Comment (victorgama) [inline bot_modules/spaces.js]: This pattern is not necessary, since the arrow function retains scope.

File: bot_modules/spaces.js [hunk 2]
```diff
@@ -1,13 +1,68 @@
...
+            response.getUser().then(u => {
+                _this.searchChannel('general')
+                    .then((c) => {
+                        c.send(`Galera, ${_this.getMentionTagForUser(u)} ${phrase} :rocket:`);
+                        response.reply('Done. :rocket:');
```
Comment (victorgama) [inline bot_modules/spaces.js]: PT-BR, to maintain consistency?

File: bot_modules/spaces.js [hunk 3]
```diff
@@ -1,13 +1,68 @@
...
+                    })
+                    .catch((ex) => {
+                        response.reply('Não consegui te colocar de spaces, o @joaomarcus não deixou. :neutral_face:');
```
Comment (victorgama) [inline bot_modules/spaces.js]: Intended to mention `joaomarcos`?

**Comment by victorgama** [comment]
Just take a look at those comments, and try to address them. Then we get this merged. ✨

**Comment by chr0nu5** [comment]
done and done!

File: bot_modules/spaces.js [hunk 4]
```diff
@@ -1,13 +1,70 @@
...
+        }
+        if (phrase !== defaultPhrase) {
+            response.getUser().then(u => {
+                this.searchChannel('general')
+                    .then((c) => {
+                        c.send(`Galera, ${this.getMentionTagForUser(u)} ${phrase} :rocket:`);
+                        response.reply('Pronto! :rocket:');
+                    })
+                    .catch((ex) => {
+                        this.searchUser('joaomarcus')
```
Comment (victorgama) [inline bot_modules/spaces.js]: Typo! His username is `joaomarcos`. 😛

**Comment by victorgama** [comment]
Just one more minor change! Pinky swear!

**Pull request merged**